editor/editor_node.cpp

```diff
@@ -2308,7 +2308,7 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			uint32_t current = editor_history.get_current();
 			Object *current_obj = current > 0 ? ObjectDB::get_instance(current) : NULL;
 
-			ERR_FAIL_COND(!current_obj->cast_to<Resource>())
+			ERR_FAIL_COND(!current_obj)
```

Comment:
Thanks for pointing it out. I'm aware of that. My point on changing these was simply to avoid segmentation faults. I'm waiting for #10581 to get merged to rescan with `cppcheck`.